camera/ios/Plugin/CameraPlugin.swift

```diff
@@ -109,19 +109,20 @@ public class CameraPlugin: CAPPlugin {
                         options.deliveryMode = .highQualityFormat
 
                         let group = DispatchGroup()
-
-                        for index in 0...(assets.count - 1) {
-                            let asset = assets.object(at: index)
-                            let fullSize = CGSize(width: asset.pixelWidth, height: asset.pixelHeight)
-
-                            group.enter()
-                            imageManager.requestImage(for: asset, targetSize: fullSize, contentMode: .default, options: options) { image, _ in
-                                guard let image = image else {
+                        if assets.count > 0 {
+                            for index in 0...(assets.count - 1) {
+                                let asset = assets.object(at: index)
+                                let fullSize = CGSize(width: asset.pixelWidth, height: asset.pixelHeight)
+
+                                group.enter()
+                                imageManager.requestImage(for: asset, targetSize: fullSize, contentMode: .default, options: options) { image, _ in
```

Comment:
We're ignoring the `info` dictionary of the `requestImage` handler, which could give us an error in the `PHImageErrorKey` key, if there is an error. I think in case of error, image is nil, so we don't have to worry about this code not working, but it might make sense to add it, or add a ticket to add it later.

Reply:
you can create a feature request ticket for returning an error in case `PHImageErrorKey` contains something, but won't be doing this in this fix PR as is not related

Reply:
Yeah - more a suggestion of future work, really. I'll create a ticket.